.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+/.vscode
```

Comment:
Please don't add editor-specific entries to `.gitignore`; users of VSCode should add this to their global git config instead.